pandas/tests/tseries/frequencies/test_freq_code.py

```diff
@@ -93,6 +93,9 @@ def test_get_to_timestamp_base(freqstr, exp_freqstr):
 @pytest.mark.parametrize(
     "freqstr,expected",
     [
+        ("A", "year"),
```

Comment:
are these tested anywhere?

Reply:
as removing the raises test

Reply:
you mean tested anywhere besides here?  the test in the previous file for period_range is effectively testing the same behavior at a higher levle

Reply:
i c, ok